lib/wash/exe/chrome.js

```diff
@@ -13,16 +13,112 @@
 // limitations under the License.
 
 import AxiomError from 'axiom/core/error';
+import Completer from 'axiom/core/completer';
 import chromeAgentClient from 'wash/chrome_agent_client';
 
 /** @typedef JsExecuteContext$$module$axiom$fs$js$execute_context */
 var JsExecuteContext;
 
+/** @const @private */
+var STDIN_ = '$STDIN';
+/** @const @private */
+var STDIN_RE_ = new RegExp(STDIN_);
```

Comment:
Good catch! In some intermediate version, it worked for me, but then the code changed and I didn't retest it. Fixed now.